### DISCUSSION TITLE
더 빠른 워크플로우를 향해 | 아카이브-로그

### DISCUSSION THREAD

**Discussion by giscus** [jongwooo/blog#351] [Comment]
# 더 빠른 워크플로우를 향해 | 아카이브-로그

오픈소스 컨트리뷰션 아카데미에서 프로젝트를 진행하면서 CI를 기다리는 시간이 오래 걸린다는 의견이 있었습니다.
이는 워크플로우가 매번 새로운 가상환경에서 실행되기 때문에, 의존성 패키지를 워크플로우마다 설치해야 해서 발생하는 문제였습니다. 비슷한 의존성을 반복적으로 설치하는 것은 네트워크 사용량을 증가시키고 런타임을 늘리는 원인이 되므로, 결국 시간과 자…

https://thearchivelog.dev/article/caching-dependencies-to-speed-up-workflows/

**Comment by Moondoyeon** [comment]
감사합니다
